packages/opal-client/requires.txt

```diff
@@ -1,5 +1,5 @@
 aiofiles
 aiohttp
-psutil==5.8.0
-tenacity==6.3.1
```

Comment:
I changed it in another PR to 8.0.1

Reply:
Updated; now the specifier is `tenacity>=8.0.1,<9`